src/dbnode/storage/shard.go

```diff
@@ -619,7 +619,10 @@ func (s *dbShard) Close() error {
 	s.state = dbShardStateClosing
 	s.Unlock()
 
-	s.insertQueue.Stop()
+	s.insertQueue.Stop() // nolint
```

Comment:
What was the lint issue here? 
In general I think it makes things more clear to indicate specific lint check in this directive, eg. `nolint: errcheck`.